failing test on dom0 yum repos

Over the past week or so, I've started seeing a failing test in the `tests/test_dom0_rpm_repo.py` checks:

```
>       assert found_lines == wanted_lines
E       AssertionError: assert ['[securedrop-workstation-dom0]',\n 'gpgcheck=1',\n 'skip_if_unavailable=False',\n 'gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test',\n 'enabled=1',\n 'baseurl=https://yum-test.securedrop.org/workstation/dom0/r$releasever',\n 'name=SecureDrop Workstation Qubes dom0 repo'] == ['[securedrop-workstation-dom0]',\n 'gpgcheck=1',\n 'skip_if_unavailable=False',\n 'gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test',\n 'enabled=1',\n 'baseurl=https://yum-test.securedrop.org/workstation/dom0/f37-nightlies/',\n 'name=SecureDrop Workstation Qubes dom0 repo']
E         At index 5 diff: 'baseurl=https://yum-test.securedrop.org/workstation/dom0/r$releasever' != 'baseurl=https://yum-test.securedrop.org/workstation/dom0/f37-nightlies/'
E         Full diff:
E           [
E            '[securedrop-workstation-dom0]',
E            'gpgcheck=1',
E            'skip_if_unavailable=False',
E            'gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test',
E            'enabled=1',
E         -  'baseurl=https://yum-test.securedrop.org/workstation/dom0/f37-nightlies/',
E         ?                                                            ^^^^^^^^^ -  ^
E         +  'baseurl=https://yum-test.securedrop.org/workstation/dom0/r$releasever',
E         ?                                                            ^^^^  + ^^^^
E            'name=SecureDrop Workstation Qubes dom0 repo',
E           ]

tests/test_dom0_rpm_repo.py:64: AssertionError
```

## Steps to reproduce

I'm able to reproduce this locally on commit 47a3065a11f2d4618f2f09f06c9f02ff9e32af50, merged this morning. The STR are:

```
make clone
make clean
# then additional cleaning, to be sure:
rpm -qa | grep '^securedrop-workstation' | xargs -r sudo dnf remove -y
find /etc/yum.repos.d -type f -iname 'securedrop-workstation*.repo' -exec sudo rm -v {} +

make dev
make test # observe failure here
```

## Initial thoughts
As the `find` command in those clean steps indicate, I'm discovering that the yum repos config on disk is maybe not as pristine as it should be:

```
[conor@dom0 ~]$ tree /etc/yum.repos.d/
/etc/yum.repos.d/
├── fedora.repo
├── fedora-updates.repo
├── qubes-contrib-dom0-r4.2.repo
├── qubes-dom0.repo
├── qubes-templates.repo
├── securedrop-workstation-dom0-dev.repo
├── securedrop-workstation-dom0.repo
└── securedrop-workstation-keyring-dev.repo
```

It looks like the `scripts/bootstrap-keyring.py` is configuring duplicate repo files, and I don't believe that should be the case.

--- a/tests/test_dom0_rpm_repo.py
+++ b/tests/test_dom0_rpm_repo.py
@@ -39,9 +39,7 @@ class SD_Dom0_Rpm_Repo_Tests(unittest.TestCase):
 
         self.config = REPO_CONFIG[self.env]
 
-    # Temporarily disabling this test pending backend fixes:
-    # https://github.com/freedomofpress/securedrop-workstation/issue/1530
-    def _test_rpm_repo_config(self):
+    def test_rpm_repo_config(self):
         repo = self.config["repo_file_name"]
         baseurl = self.config["yum_repo_url"]
         repo_file = f"/etc/yum.repos.d/{repo}"

